eng/scripts/Get-TypeSpec-Folders.ps1

```diff
@@ -9,8 +9,9 @@ param (
 )
 
 $tspFiles = @()
+$osSlash = [IO.Path]::DirectorySeparatorChar
 if ([string]::IsNullOrEmpty($TargetBranch) -or [string]::IsNullOrEmpty($SourceBranch)) {
-  $tspFiles = (Get-ChildItem -path ./specification tspconfig.yaml -Recurse).FullName -replace "$($pwd.Path)/"
+  $tspFiles = (Get-ChildItem -path ./specification tspconfig.yaml -Recurse).FullName -replace [Regex]::Escape("$($pwd.Path)$osSlash")
```

Comment:
What is the goal here? Is it to get the relative path? If so you can also use the Path APIs for this https://learn.microsoft.com/en-us/dotnet/api/system.io.path.getrelativepath?view=net-7.0

Reply:
To avoid the slash issue in all the places I suggest you simply replacing "\" with "/" the forward. Windows will also work with forward slash paths as well.